### PR TITLE
Bump version for Fable.Cli, Fable.Core, mocha, fable-publish-utils

### PR DESCRIPTION
**Pull request by delneg**
Added missing "postinstall" script for dotnet tool restore
Also deleted fable-compiler-js, dunno what it's for, tests ran fine

### CODE REVIEW

**Comment by MangelMaxime** [comment]
Hello `fable-compiler-js` was used to execute the `build.fsx` file which is a Fable application.

But I think with Fable 3 we can use `dotnet fable build.fsx --run publish` or something like that.

File: src/Fable.Promise.fsproj [hunk 1]
```diff
@@ -14,6 +14,6 @@
     <Content Include="*.fsproj; *.fs" PackagePath="fable\" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Fable.Core" Version="3.1.5" />
+    <PackageReference Include="Fable.Core" Version="3.2.3" />
```
Comment (alfonsogarciacaro) [inline src/Fable.Promise.fsproj]: Same comment as in https://github.com/fable-compiler/fable-fetch/pull/15#discussion_r559281122

**Comment by alfonsogarciacaro** [comment]
Yes, that's right. As @MangelMaxime says, using something like `dotnet fable build.fsx --runScript publish` should work (note the argument is `--runScript`, see https://github.com/fable-compiler/Fable/issues/2335). Although lately I made the publish utils compatible with .NET so it's more convenient to just use F# interactive: `dotnet fsi build.fsx publish` (I'm still using npm for the distribution, because it's much simpler than Nuget to package source files).

**Comment by delneg** [comment]
Fixed according to comments above

**Comment by alfonsogarciacaro** [comment]
Thanks @delneg!

**Pull request merged**